src/_components/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+import { Theme } from '../../_styles/theme';
+
+import Icon from '../icon/Icon';
+
+import * as Styled from './style';
+
+type TProps = {
+  isChecked: boolean;
+  label?: React.ReactNode;
+  onChange?: (value: boolean, name?: string) => void;
+  name?: string;
+  checkedColor?: keyof Theme['colors'];
+  iconSize?: number | 'small' | 'large';
```

Comment:
Since this is directly passed to the `Icon` component, why not reference the Icon's `size` prop here?